multi_asset/multi_asset.md

```diff
@@ -266,3 +274,32 @@ implement the `multi_token_receiver`.
 
 Only non-standard transfer functions MAY allow tokens to be sent to a recipient
 contract that does NOT implement the necessary `multi_token_receiver` hook functions.
+
+#### Token owner contract implementation guidelines
+
+This specification focuses on token transfer logic only. Implementation of the
+actual token receiver contract may differ depending on the particular business
+use-case. However, potential locking of tokens on the receiver account must be
+taken into consideration.
+
+Token owner contract MUST implement `multi_token_receiver` interface to receive
+tokens.
+
+By default, transfer **from** a token owner to another contract can be
+initiated only by the owner itself. If owner contract implementation does
+not allow to initiate such transfer and/or add operators (which also can be
+performed only by the owner), tokens will remain locked on such contract forever.
+
+There are a few possible ways to enable transferring tokens from a token owner
+contract. The concrete implementation of multi asset contract and/or owner
+contract may use the following strategies or their combination:
+
+  1. Add administrative entry points to multi asset contract to allow burn
+  tokens for any token owner.
+  2. Implement forwarding token receiver contract. `On_multi_tokens_received`
+  implementation should initiate another transfer operation which will forward
+  all received tokens to another owner.
```

Comment:
Forwarding alone doesn't seem to solve anything since now tokens may end up locked forever in that other token owner contract.

Reply:
This strategy can be used in combination with others. It solves the problem for the forwarder contract

Reply:
yes, but it is listed as a standalone, independent option, which it is not

Reply:
Right from above: “The concrete implementation of multi asset contract and/or owner
contract may use the following strategies **or their combination**:”